common/endpoints/endpoints.service.ts

```diff
@@ -50,9 +50,10 @@ export class HsCommonEndpointsService {
             user: ds.user,
             liferayProtocol: ds.liferayProtocol,
             originalConfiguredUser: ds.user,
-            getCurrentUserIfNeeded:
-              HsCommonLaymanService.getCurrentUserIfNeeded,
+            getCurrentUserIfNeeded: () =>
+              HsCommonLaymanService.getCurrentUserIfNeeded(tmp),
```

Comment:
What is this trick? Some nasty 'this' context again?

Reply:
Pretty much only this time I knew before that it would fail, since I'm passing the function inside the array element/object, but the function itself resides inside the class.